react/react1/week3/todo-list/src/App.js

```diff
@@ -0,0 +1,64 @@
+import ToDoList from "./components/ToDoList";
+import { Timer } from "./components/Timer";
+import { useState, useEffect } from "react";
+import { AddForm } from "./components/AddForm";
+import "./App.css";
+
+const fetchUrl =
+  "https://gist.githubusercontent.com/benna100/391eee7a119b50bd2c5960ab51622532/raw";
+
+function App() {
+  const [todos, setTodos] = useState([]);
+
+  //Fetching todos from API on window load
+  useEffect(() => {
+    addTodosFromAPI(fetchUrl);
+  }, []);
+
+  async function addTodosFromAPI(fetchUrl) {
+    const response = await fetch(fetchUrl);
+    const fetchedTodos = await response.json();
+    setTodos(fetchedTodos);
+  }
+
+  //Actions with the list
+  //updateDescription is going to be passed to the ToDoItem component
+  const updateDescription = (e, description) => {
+    const todoToUpdate = e.target.parentNode.id;
```

Comment:
if you have access to the id where this function is being called maybe you could just send it as a param instead of going through the dom?